core/acl.py

```diff
@@ -113,3 +119,34 @@ async def repeat(self, ctx, *, input: str):
 
     # user's roles are not mapped to any ACL group, return default
     return rule.default
+
+
+def version(major: int, minor: int, micro: Optional[int] = None):
```

Comment:
Return type would be nice. Just `) -> bool:`.

Reply:
It does not always return `bool`, function is returned when version is correct

Reply:
Oh, I see, this is not implemented to be used inside of `commands.check()` decorator. I didn't think of this solution. Well, we'll see how it works, we can always update it if it does not work as intended before the 1.0 release sometime in the future.

In this case the return type should be `Union[bool, Callable]`. Or would returning `None` be better solution to insufficient version?

Reply:
I think returning `None` makes more  sense as it is not a check that something is correct but the calling of the function/method depends on this, returning `None` means that it had nothing to call.